### DISCUSSION TITLE
Request -Multiple Sub folders

### DISCUSSION THREAD

**Discussion by Rhodynoliaeth** [BlafKing/sd-civitai-browser-plus#137] [Ideas]
Hi, I use several SSDs because the models take up a lot of space. I would like to be able to add in the options several directory paths to save Loras and Checkpoints, etc. And to be able to choose between these folders in the search and download page. Or maybe have a history of recently used folders.

Thank you for your attention!

**Comment by BlafKing** [comment]
Thanks for the suggestion! Unfortunately, I don't think this would be a great feature.
Since, when you use a custom path, SD-WebUI will then not detect any files put inside that path.

So if you save a checkpoint/Lora to a custom path, then you won't be able to use them in the checkpoint dropdown/Lora menu of SD-WebUI and this extension will then also not detect them as installed.

I think a better solution would be to do one of the following options:

* Use [SD-WebUI's command arguments](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Command-Line-Arguments-and-Settings) to set any content-type to a custom folder, for example: `--ckpt-dir C:/Checkpoints`

* An other solution (which I also use) could be to use [symbolic links](https://www.howtogeek.com/16226/complete-guide-to-symbolic-links-symlinks-on-windows-or-linux/), this way, you can place any models inside the LORA or checkpoint directory while it's actually being stored on another drive.
You can then select that folder as a sub folder when downloading any models, and then it will get stored on another drive.
‎
If you're on windows, I recommend to use a tool called [Link Shell Extension](https://schinagl.priv.at/nt/hardlinkshellext/linkshellextension.html#download)
Here's an example of how you could use this:


https://github.com/BlafKing/sd-civitai-browser-plus/assets/9644716/4068d870-3cac-4672-a812-f1043083b550
